src/Mod/TechDraw/Gui/ViewProviderProjGroup.cpp

```diff
@@ -193,7 +193,7 @@ bool ViewProviderProjGroup::onDelete(const std::vector<std::string> &)
     // if there are section or detail views we cannot delete because this would break them
     if (!ViewList.empty()) {
         bodyMessageStream << qApp->translate("Std_Delete",
-            "The group cannot be deleted because its items have the following\nsection or detail views, or leader lines that would get broken:\n");

```

Comment:
This one must be kept. I added this purposely because a list of objects follow. The list needs a newline before because it doesn't have one at the beginning.

Reply:
can the newline be added outside of the translation?

Reply:
> can the newline be added outside of the translation?

Not out of the box because it is not known in advance what will be in the list. But why is it so bad to have a newline in a translatable string?

Reply:
@donovaly because translators miss it sometimes. So removing the newline makes sure they don't. 

Can't we have logic that says, if there is a list to print out, prefix it with a newline ?

Reply:
> Can't we have logic that says, if there is a list to print out, prefix it with a newline ?

OK.
I had a closer look and this way also find a logical issue. This PR fixes the trailing newline and the logic issue: https://github.com/FreeCAD/FreeCAD/pull/3744

Could you please update this PR that it doesn't contain the ViewProviderProjGroup.cpp ? (to assure both PR can be merged without problems)